caffe2/contrib/tensorrt/tensorrt_tranformer.cc

```diff
@@ -1,11 +1,13 @@
 #include "caffe2/contrib/tensorrt/tensorrt_tranformer.h"
-#include <NvInfer.h>
-#include <onnx2trt.hpp>
 #include "caffe2/contrib/tensorrt/trt_utils.h"
+#include "caffe2/core/context_gpu.h"
 #include "caffe2/core/logging.h"
 #include "caffe2/core/operator.h"
 #include "caffe2/onnx/onnx_exporter.h"
 
+#include <NvInfer.h>
```

Comment:
I'd suggest move such include with "<>" in front of includes with """".
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes